srcpkgs/snes9x/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'snes9x'
+pkgname=snes9x
+version=1.56.2
+revision=1
+build_wrksrc="unix"
+build_style=gnu-configure
+configure_args="--enable-netplay"
+hostmakedepends="intltool nasm autogen autoconf automake"
+makedepends="minizip-devel pulseaudio-devel libpng-devel SDL-devel SDL2-devel libXv-devel
+ xorg-server-devel zlib-devel gtk+3-devel"
+depends="minizip libpulseaudio pulseaudio desktop-file-utils libpng SDL SDL2 adwaita-icon-theme
+ libXv gtk+3 zlib"
+short_desc="A Super Nintendo emulator in active development"
+maintainer="VladTheImpaler <vladimir-romanov-ii@yandex.com>"
+license="LGPL"
```

Comment:
not a spdx short-identifier

Reply:
Okay, I will look up correct one, but this one is currently in use with snes9x-gtk package which is why is there